src/item.cpp

```diff
@@ -1395,6 +1395,10 @@ std::string item::info( bool showtext, std::vector<iteminfo> &info ) const
             info.push_back( iteminfo( "DESCRIPTION",
                                       _( "* This piece of clothing is large enough to accommodate <info>mutated anatomy</info>." ) ) );
         }
+        if( is_armor() && has_flag( "ALLOWS_NATURAL_ATTACKS" ) ) {
+            info.push_back( iteminfo( "DESCRIPTION",
+                                      _( "* This piece of clothing won't special attacks that involve <info>mutated anatomy</info>." ) ) );
```

Comment:
The string reads odd. "This piece ... won't special attacks ...", isn't there another verb missing?